routers/user/home.go

```diff
@@ -149,6 +151,9 @@ func Dashboard(ctx *context.Context) {
 	ctx.HTML(200, tplDashboard)
 }
 
+// Regexp for repos query
+var issueReposQueryPattern = regexp.MustCompile(`^\[(\d+,{0,1})+\]$`)
```

Comment:
```suggestion
var issueReposQueryPattern = regexp.MustCompile(`^\[\d+(,\d+)*\]$`)
```

Reply:
there exist two paterns:
`[2,6,3]` and  `[2,6,3,]`

Reply:
this is because of template range methode

Reply:
Oh in that case:

```suggestion
var issueReposQueryPattern = regexp.MustCompile(`^\[\d+(,\d+)*,?\]$`)
```

Is still less complex than the original one.